src/theme/popover/ods-chart-popover-definitions.ts

```diff
@@ -268,10 +304,12 @@ class BOOSTED4_Definition extends ODSChartsPopoverDefinitionWithRenderer {
 }
 
 /**
+ * `ODSChartsPopoverManagers` is used to specify popoverDefinition parameter (popup/tooltip renderer) of {@link ODSChartsTheme.externalizePopover}.
+ *
  * Available popover/tooltip renderer:
- * - ODSChartsPopoverManagers.NONE: to use default ECharts template to externalize tooltip/popover HTML element, implementing Orange Design System
- * - ODSChartsPopoverManagers.BOOSTED5: to use Boosted 5 tooltip/popover
- * - ODSChartsPopoverManagers.BOOSTED4: to use Boosted 4 tooltip/popover
+ * - `ODSChartsPopoverManagers.NONE`: to use default ECharts template to externalize tooltip/popover HTML element, implementing Orange Design System
+ * - `ODSChartsPopoverManagers.BOOSTED5`: to use Boosted 5 tooltip/popover
+ * - `ODSChartsPopoverManagers.BOOSTED4`: to use Boosted 4 tooltip/popover
```

Comment:
Add a warning that using Boosted 4 or Boosted 5 rendering requires dependency on the boosted library and access to the boosted global variable.